### PR TITLE
0033762: Documentation - Fix various typos found in codebase

### PR DESCRIPTION
**Pull request by luzpaz**
No attribution necessary. Please feel free to merge.

### CODE REVIEW

**Comment by dpasukhi** [comment]
Issue: https://tracker.dev.opencascade.org/view.php?id=33762

File: src/BOPTools/BOPTools_AlgoTools.cxx
Comment (dpasukhi) [inline src/BOPTools/BOPTools_AlgoTools.cxx:1927]: @luzpaz Could you remove please that line?
(or probably it is fine. Well, during today i will wait, after i will merge with IR)

**Review by dpasukhi — APPROVED** [review]
Approved. But if one line will be modified. It will be nice :)

**Comment by luzpaz** [comment]
Sorry coming to this a little late. Thanks for taking care of it.

**Comment by dpasukhi** [comment]
Thank you for your work!
My apologies, that I don't wait for your feedback, but it was just a little misprint :)

**Pull request merged**
